stablehlo/transforms/ChloDecompositionPatternsMath.td

```diff
@@ -643,17 +643,6 @@ def : Pat<(CHLO_AcosOp ComplexElementType:$z),
 //
 //       1 - x * x == (1 - x) * (1 + x)
 //
```

Comment:
The comment should be removed in full:
```suggestion
```

Since ChloDecompositionPatternsMath.td is generated, editing the file directly is not recommended. However, in this particular case, I think it is OK as a temporary solution. I'll address this as a first thing when returning from my vacation.